src/editor/model/Editor.ts

```diff
@@ -975,7 +961,7 @@ export default class EditorModel extends Model {
     // @ts-ignore
     const { editors = [] } = config.grapesjs || {};
     const shallow = this.get('shallow');
-    clearTimeout(this._storageTimeout);
+    this._storageTimeout ?? clearTimeout(this._storageTimeout);
```

Comment:
I don't know if those TS issue you see (not sure why I'm don't see them on my side) are related to this part but with this change, by using `??` it looks like it won't be executed it case `_storageTimeout` is defined, did you mean to use `&&`?

Reply:
And yes I wanted to use &&.